examples/glamorous-jest-react/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+/* eslint-disable */
+import * as glamorous from 'glamorous';
```

Comment:
`glamorous` exports itself as the `default` export, so this should be `import glamorous from 'glamorous'`